src/commands/dev/mod.rs

```diff
@@ -19,12 +21,19 @@ pub enum DevSubcommand {
         #[arg(default_value = ".")]
         path: PathBuf,
     },
+    /// Edit the config file from the vault
```

Comment:
According to the RFC, the help message for this command should be "Allow to view the current secrets file from Bunker and make the necessary changes."

```suggestion
    /// Allow to view the current secrets file from Bunker and make the necessary changes.
```